aws/src/main/java/org/apache/iceberg/aws/s3/S3OutputStream.java

```diff
@@ -86,7 +86,8 @@
   private boolean closed = false;
 
   @SuppressWarnings("StaticAssignmentInConstructor")
-  S3OutputStream(S3Client s3, S3URI location, AwsProperties awsProperties) throws IOException {
+  S3OutputStream(S3Client s3, S3URI location, AwsProperties awsProperties)
+      throws IOException, SecurityException {
```

Comment:
same comment as below, remove SecurityException after throws

Reply:
Removed it.